source/process/bug-fix-release.md

```diff
@@ -84,12 +85,8 @@ Review the [Release Features & Bugs Quality Gate Guidelines](https://docs.google
 1. Release Manager:
     - Post this checklist in Release Checklist channel
     - Verify all items in the last posted release checklist are complete
-    - Update the GitHub meta issue:
-       - Include a link to the Changelog on the documentation branch
-       - Post comments to the meta issue with approved fixes for the next RCs
-       - Update download links and testing server links to the latest RCs
-    - After build is cut, tweet announcement that RC1 is ready (see [example](https://community.mattermost.com/core/pl/tefx1ijyz7bs8mabuxmpq9f7pw))
     - Generate a list of contributors for Changelog
+    - Ask Platform team to provide list of [database changes](https://github.com/mattermost/mattermost-server/blob/master/store/sqlstore/upgrade.go), [API changes](https://github.com/mattermost/mattermost-server/commits/master/model/client.go), and [WebSocket event changes](https://github.com/mattermost/mattermost-server/blob/master/model/websocket_message.go#L13) for Changelog
```

Comment:
For platform team regarding database/API/... updates - do you ask a specific dev, dev lead, or the team in general? Do you have an example post to link to?